x-pack/plugins/canvas/public/components/layout_annotations/layout_annotations.scss

```diff
@@ -0,0 +1,81 @@
+.canvasAlignmentGuide {
+  transform-origin: center center; /* the default, only for clarity */
+  transform-style: preserve-3d;
+}
+
+.canvasBorderConnection {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+  border-top: 1px dashed $euiColorLightShade;
+  border-left: 1px dashed $euiColorLightShade;
+}
+
+.canvasBorderResizeHandle {
+  @include euiSlightShadow;
+  transform-origin: center center; /* the default, only for clarity */
+  transform-style: preserve-3d;
+  display: block;
+  position: absolute;
+  height: 8px;
```

Comment:
These should use EUI vars, but I realize you're just migrating stuff as part of a tech debt clean up. Something to clean up in the future.

Reply:
Fixed in the latest update.